types/node/http.d.ts

```diff
@@ -77,6 +77,10 @@ declare module "http" {
         defaultPort?: number | string;
         localAddress?: string;
         socketPath?: string;
+        /**
+         * @default 8192
+         */
+        maxHeaderSize?: number;
```

Comment:
```suggestion
        readonly maxHeaderSize: number;
```

Reply:
These are options, not sure why these would be readonly?

Reply:
Ah sorry. You are correct. I mixed this up with the module static which is not part of this PR.